index.html

```diff
@@ -152,6 +152,20 @@ <h1 class="lg">Skills</h1>
          <hr class="divider2">
         </div>
       </section>
+      <footer class="contact">
+        <h2 class="contact-me"><b>Contact me</b></h2>
+        <p class="more">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mrgvqyqz">
+          <label for="full-name">
```

Comment:
- [ ] Form uses the `action` and `method` attributes. The `method` value equals `POST`. Please implement it.